cron.sh

```diff
@@ -2,8 +2,8 @@
 set -euo pipefail # See http://redsymbol.net/articles/unofficial-bash-strict-mode/
 IFS=$'\n\t'
 
-# Execute via crontab by sol@quaoar1:
-# 00 9 * * * cd /home/sol/git/mabxml-elasticsearch ; bash cron.sh >> logs/cron.sh.log 2>&1
+# Execute via crontab by hduser@weywot1:
+# 30 6 * * * ssh sol@quaoar1 "cd /home/sol/git/mabxml-elasticsearch ; bash -x cron.sh >> logs/cron.sh.log 2>&1"
 
```

Comment:
This makes one big file instead of one file per action. May be ok if log output is not too big and so searching for the day of interest is not to inconvenient, though. Is this so?